python/tank/pipelineconfig_factory.py

```diff
@@ -538,10 +538,17 @@ def _get_pipeline_configs_for_path(path, data):
     for pc in data["pipeline_configurations"]:
 
         for s in storages:
-            # all pipeline configurations are associated
+
+            # all installed/classic pipeline configurations are associated
```

Comment:
nit: this should say something like "projects not using the templates/schema system may not have a project.Project.tank_name" set"

Reply:
addressed